fix GetIndicatorDBotScore IPv6 mapping bug (#17159)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_6_24.md

#### Scripts
##### GetIndicatorDBotScore

> - Fixed an issue where IPv6 indicator type was not mapped at all to a DBotScoreType.

Reviewer: ```suggestion
- Fixed an issue where IPv6 indicator types were not mapped to a DBotScoreType.
```

Author: done
